docs/configuring/enabling-nio.html

```diff
@@ -0,0 +1,127 @@
+          <li class="nested-sidebar-item active"><a href="../configuring/enabling-nio.html">Enabling SwiftNIO</a></li>
+        </ul>
+      </ul>
+    </aside>
+    <div id="burgerIcon" class="burger-icon" onclick="showSidebar()">
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+    </div>
+    <div class="docs-item-2 search-container">
+    </div>
+    <nav class="docs-item-3 docs-nav">
+      <button id="api-button" class="apiref-button" type="button" name="button" onclick="window.open('https://ibm-swift.github.io/Kitura/')">API Reference</button>
+      <a class="nav-item" target="_blank" href="http://slack.kitura.io/">Need help?</a>
+      <a class="nav-item" target="_blank" href="https://github.com/IBM-Swift/kitura.io/issues">Found an issue?</a>
+    </nav>
+    <div id="doc-container" class="docs-item-4 docs-window">
+      <main>
+        <h1 class="heading-1">Enabling SwiftNIO</h1>
+        <p>In February 2018, Apple announced <a href="https://github.com/apple/swift-nio">SwiftNIO</a> — a new cross-platform, asynchronous, event-driven networking framework for building servers and clients in Swift. SwiftNIO has the potential to provide all the low-level socket-handling and HTTP protocol support in Kitura, and hence could replace the current <a href="https://github.com/IBM-Swift/Kitura-net">Kitura-net</a> and <a href="https://github.com/IBM-Swift/BlueSocket">BlueSocket</a> libraries which underpin Kitura today. The Kitura team released <a href="https://github.com/IBM-Swift/Kitura-NIO">Kitura-NIO</a>, a Kitura interface to SwiftNIO in May 2018.</p>
+        <p>Kitura-NIO has been designed to be API-compatible with the current Kitura-net API, though the underlying implementation uses SwiftNIO.</p>
+        <p>The major reason behind the decision to keep the API compatible with Kitura-net was API stability. We wanted Kitura users to experiment with Kitura-NIO, without needing to make any code changes to their Kitura applications.</p>
+        <p>Kitura-NIO fully supports HTTP and HTTPS, both servers and clients. It also has IPv6 support. A SwiftNIO based implementation of Kitura-WebSocket (<a href="https://github.com/IBM-Swift/Kitura-WebSocket-NIO">Kitura-WebSocket-NIO</a>) was released in April 2019, as part of <a href="https://developer.ibm.com/swift/2019/04/17/announcing-kitura-2-7-and-more/">Kitura 2.7</a>.</p>
+        <p>It is our intent to make Kitura-NIO the default networking stack in a future release of Kitura.</p>
+        <div class="underline"></div>
+        <h2 class="heading-2">Using Kitura-NIO</h2>
+        <p>With Kitura 2.5 and future releases, to run on top of Kitura-NIO (instead of the current default Kitura-net) all you need to do is set an environment variable called <strong>KITURA_NIO</strong> before building your Kitura application:</p>
+        <pre><code>export KITURA_NIO=1 && swift build
```

Comment:
Think this should be `swift build -DKITURA_NIO=1`??

Reply:
@ianpartridge `swift build -DKITURA_NIO=1`
gives me 
error: unknown option -DKITURA_NIO; use --help to list available option

I did check `export KITURA_NIO=1 && swift build` and that works fine (and is what Pushkar documented in the Kitura-NIO README).